rrmngmnt/host.py

```diff
@@ -336,3 +337,41 @@ def is_connective(self, tcp_timeout=20.0):
             ret = False
 
         return ret
+
+    def download_file_from_url(self, url, f_dir):
```

Comment:
I will rename function
About wget module, I believe I has some problem with our framework:
Traceback (most recent call last):
  File "/home/alukiano/PycharmProjects/ART/art/test_handler/plmanagement/plugins/unittest_test_runner_plugin.py", line 209, in __call__
    self.f()
  File "/home/alukiano/PycharmProjects/ART/art/tests/rhevm/unittests/local_test/local_test.py", line 40, in test01_create_vm
    filename = wget.download(url=url, out="/tmp")
  File "/usr/lib/python2.7/site-packages/wget.py", line 526, in download
    (tmpfile, headers) = ulib.urlretrieve(binurl, tmpfile, callback)
  File "/usr/lib64/python2.7/urllib.py", line 98, in urlretrieve
    return opener.retrieve(url, filename, reporthook, data)
  File "/usr/lib64/python2.7/urllib.py", line 271, in retrieve
    reporthook(blocknum, bs, size)
  File "/usr/lib/python2.7/site-packages/wget.py", line 513, in callback_charged
    callback_progress(blocks, block_size, total_size, bar_function=bar)
  File "/usr/lib/python2.7/site-packages/wget.py", line 461, in callback_progress
    width = min(100, get_console_width())
  File "/usr/lib/python2.7/site-packages/wget.py", line 337, in get_console_width
    ioctl(sys.stdout.fileno(), TIOCGWINSZ, winsize)
AttributeError: StringIO instance has no attribute 'fileno'
and I do not want to spend time on investigation)